rma/wizard/stock_picking_return.py

```diff
@@ -129,6 +124,10 @@ def create_returns(self):
                 )
             vals_list = self._prepare_rma_vals_list()
             rmas = self.env["rma"].create(vals_list)
+            proc_group_vals = rmas[:1]._prepare_procurement_group_vals()
+            proc_group_vals["name"] = self.picking_id.name
+            proc_group = self.env["procurement.group"].create(proc_group_vals)
+            rmas.write({"procurement_group_id": proc_group.id})
```

Comment:
Hi @TheClaud99 ,

IMO, the issue is due to the non-unified of procurement groups creation.

When the picking return wizard is used, one procurement group is created for all RMAs. However, this is not the case when the sales order return wizard is used or when RMAs are created manually.

#423 addresses this issue by unifying the procurement group creation at RMA confirmation, regardless of the RMA origin.

I propose you revert these changes and keep the fix of the method name in rma_sale, 

what do you think?

Reply:
Hi @sbejaoui, I think it should be ok in your way as well (with also the advantage of having procurement.group creation unified), I can completely revert my commit 5f73bf6b5e962a30502b3d802c91142b56aae0f9 and wait for your pr to be merged